now.json

```diff
@@ -0,0 +1,12 @@
+{
+  "version": 2,
+  "builds": [
+    {
+      "src": ".build-for-now.sh",
```

Comment:
bummer that you have to both manually specify the script *and* put it in the root 😆